airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```diff
@@ -75,6 +77,40 @@ def build_refresh_request_body(self) -> Mapping[str, Any]:
 
         return payload
 
+    @property
+    def refresh_token_error_status_codes(self) -> Tuple[int, ...]:
```

Comment:
nit: can we pass these 3 parameters to to __init__ instead of defining public properties? 

The benefits would be:
1. The behavior is much easier to understand since we don't need to go through a long chain of inheritance
2. This will be easier to extend as our needs change since the implementation details are hidden in the class

Reply:
Thanks for the advice. Could you please verify the update?